btf/btf.go

```diff
@@ -37,7 +37,7 @@ type Spec struct {
 
 	// All types contained by the spec. For the base type, the position of
 	// a type in the slice is its ID.
-	types types
+	types []Type
```

Comment:
How come this was changed?

Reply:
Because the only function on `types` is ByID, which doesn't take split BTF into account. `types` should be removed but that can happen in a follow up.